src/components/Dashboard/Dashboard.test.tsx

```diff
@@ -3,20 +3,22 @@ import { fireEvent, render, screen, waitFor } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
```

Comment:
Not for now, but I think these test class is too big. Perhaps it could be split by functionality or something similar on next changes.